src/java.base/share/classes/sun/security/util/math/intpoly/IntegerPolynomial.java

```diff
@@ -329,9 +329,10 @@ private void setLimbsValue(BigInteger v, long[] limbs) {
     }
 
     protected void setLimbsValuePositive(BigInteger v, long[] limbs) {
-        BigInteger mod = BigInteger.valueOf(1 << bitsPerLimb);
+        assert bitsPerLimb < 32;
```

Comment:
I may have this assert in the constructors as it is a final field.

Reply:
I added it here as a comment explaining why using intValue is okay here; it wouldn't serve its purpose if I moved it elsewhere.
Asserts are eliminated by the compiler by default anyway. If you think this assert shouldn't be here, I can replace it with a normal comment.

Reply:
The assert feature can be turned on.  I may prefer to use comment if that's the intention.